### PR TITLE
CommonScripts last PR

### PR DESCRIPTION
**Pull request by samuelFain**

### CODE REVIEW

**Comment by daryakoval** [comment]
DO NOT MERGE with my debug comments.

**Comment by daryakoval** [comment]
Hi, Please make sure that there are no leftovers from my debugging. 

After a lot of debugging, we have a bug that won't fix, because we led to some irrational installation of packs that caused this bug. 
We decided to do some architectural changes to solve the issue. It will take some time to solve it.

Please remove from this pr the scripts that use rasterize and Whois. (The scripts that caused CommonScripts to be dependent on rasterize and Whois), and move them to another PR that will be blocked. 
Check if the original PR's build passes without the scripts I asked to remove. 
Update me if any assistance is needed.

**Review by Shellyber — COMMENTED** [review]
Please fix the conflicts as well

File: Packs/CommonScripts/pack_metadata.json [hunk 1]
```diff
@@ -47,6 +47,14 @@
         "RemoteAccess": {
             "mandatory": false,
             "display_name": "RemoteAccess v2"
+        },
+        "rasterize": {
```
[the remark this answers is not in the document]
Comment (maimorag) [reply Packs/CommonScripts/pack_metadata.json]: somehow it didn't cause a problem in the build, but I removed it from here since the script that uses it isn't here

File: Packs/CommonScripts/.pack-ignore [hunk 2]
```diff
@@ -82,8 +82,26 @@ ignore=SC105
 [file:LoadJSONFileToContext.yml]
 ignore=SC105
 
+[file:ExportContextToJSONFile.yml]
```
Comment (Shellyber) [inline Packs/CommonScripts/.pack-ignore:85]: From where is all this coming?

File: Packs/CommonScripts/ReleaseNotes/1_11_14.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
 
 #### Scripts
 ##### GetTime
-- Documentation and metadata improvements.
\ No newline at end of file
+- Documentation and metadata improvements.
```
Comment (Shellyber) [inline Packs/CommonScripts/ReleaseNotes/1_11_14.md:4]: This file shouldn't be here, please revert it from master

File: Packs/CommonScripts/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py [hunk 4]
```diff
@@ -258,13 +258,13 @@ def fetch_indicators_from_file(args):
     return human_readable, domain_context, indicator_list
 
 
-def main():
+def main():  # pragma: no cover
```
Comment (Shellyber) [inline Packs/CommonScripts/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py:261]: Why did you add here the no cover?

File: Packs/CommonScripts/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py [hunk 5]
```diff
@@ -258,13 +258,13 @@ def fetch_indicators_from_file(args):
...
     try:
         return_outputs(*fetch_indicators_from_file(demisto.args()))
     except Exception as ex:
         return_error('Failed to execute Fetch Indicators From File. Error: {}'.format(str(ex)),
                      error=traceback.format_exc())
 
 
-if __name__ in ('__main__', '__builtin__', 'builtins'):
+if __name__ in ('__main__', '__builtin__', 'builtins'):  # pragma: no cover
```
Comment (Shellyber) [inline Packs/CommonScripts/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py:269]:
```suggestion
if __name__ in ('__main__', '__builtin__', 'builtins'):
```

File: Packs/CommonScripts/Scripts/ParseYAML/ParseYAML.py [hunk 6]
```diff
@@ -49,9 +49,9 @@ def yamlload(args: Dict[str, Any]) -> CommandResults:
     )
 
 
-def main():
+def main():  # pragma: no cover
```
Comment (Shellyber) [inline Packs/CommonScripts/Scripts/ParseYAML/ParseYAML.py:52]: @samuelFain Why the no cover here?

**Review by Shellyber — COMMENTED** [review]
@maimorag @samuelFain See my notes

File: Packs/CommonScripts/Scripts/ParseYAML/ParseYAML.py (hunk 6 above)
Comment (samuelFain) [reply Packs/CommonScripts/Scripts/ParseYAML/ParseYAML.py:52]: Since the logic is outside the main function i did to get more coverage, but because to current coverage for ParseYAML is already 100% i will remove it.

File: Packs/CommonScripts/ReleaseNotes/1_11_14.md (hunk 3 above)
Comment (samuelFain) [reply Packs/CommonScripts/ReleaseNotes/1_11_14.md]: Done.

File: Packs/CommonScripts/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py (hunk 4 above)
Comment (maimorag) [reply Packs/CommonScripts/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py]: added more tests, not sure why pragma: no cover anymore

File: Packs/CommonScripts/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py (hunk 5 above)
Comment (maimorag) [reply Packs/CommonScripts/Scripts/FetchIndicatorsFromFile/FetchIndicatorsFromFile.py]: tests added

File: Packs/CommonScripts/Scripts/ParseYAML/ParseYAML.py (hunk 6 above)
Comment (maimorag) [reply Packs/CommonScripts/Scripts/ParseYAML/ParseYAML.py]: putting resolve since Shmuel removed not cover

**Review by Shellyber — APPROVED** [review]
